src/sql/plot.py

```diff
@@ -607,6 +618,19 @@ def _histogram_stacked(
     return data
 
 
+@modify_exceptions
+def _filter_aggregate(filter_queries):
+    """Return a single filter query based on multiple queries"""
+    final_filter = ""
+    for query in filter_queries:
+        if query is not None:
+            if query == filter_queries[0]:
+                final_filter += f"WHERE {query}"
```

Comment:
I'm not sure I understand this part.

What will happen in the following case:

```
query_a = "SELECT * FROM a"
query_b = "SELECT * FROM b"
query_c = "SELECT * FROM a"
```

Reply:
The `_filter_aggregate` only combines the filters we define for the template. 
eg.
1) WHERE `colums` == NULL
2) WHERE `column' == `value`

Earlier we had only 1 facet filter, but now adding another filter to skip NULL rows. 
This function combines them into 1 filter. 

We won't be using it in for aggregating different queries.